README.md

```diff
@@ -41,7 +41,13 @@ Using bower:
 $ bower install axios
 ```
 
-Using cdn:
+Using jsDelivr CDN:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/axios@0.18.0/dist/axios.min.js" integrity="sha256-mpnrJ5DpEZZkwkE1ZgkEQQJW/46CSEh/STrZKOB/qoM=" crossorigin="anonymous"></script>
```

Comment:
By constraining to the version, its something else that has to be updated on every release, do we need this?

Reply:
FWIW SRI is an important security standard (just ask [@KingstonTime](https://twitter.com/@KingstonTime) @metromoxie & @freddyb) that [Bootstrap](http://getbootstrap.com/docs/4.1/getting-started/introduction/#quick-start) updates each release.

Also, it is also horrible practice for those to use the `latest` version which most green developers do and leave until a breaking change happens. They then go to Twitter, GitHub Issues and/or SO to complain how their site broke because...you get the point ;)

Reply:
It is, admittedly, a tricky balance, since you want to ensure that users are getting what they expect from the CDN, but you want to make sure that new users are getting up-to-date versions. I do think that Bootstrap strikes a good balance here by automating the <script> tag updates.

Additionally, if the concern is security fixes, I've heard of folks placing a warning in the console to alert devs of new versions and suggesting they upgrade. Per @jdorfman's point, it's always "fun" to have a "latest" version swap out from underneath you to a breaking change, so better to encourage folks to move to the newest version.

Reply:
I think if this link has a version, then the unpkg link should also include the version. It's odd if they are not linking to the same code.

Reply:
I agree with @styfle @Jamesking56 any thoughts?

Reply:
Agree with @styfle but we need an automated way to update the script tag per release then